knack/log.py

```diff
@@ -126,11 +126,16 @@ def configure(self, args):
         :param args: The arguments from the command line
         :type args: list
         """
+        root_logger = logging.getLogger()
+
+        if root_logger.handlers:
+            # handlers already configured
+            return
```

Comment:
When run in `pytest` environment, `root_logger` is already set by `pytest`. If we don't return here before `cli_logger.propagate = False`, `cli_logger.propagate = False` will take effect and intercepts all logs sent to `cli_loggers`.

With this change, all logs will be handled by `pytest` directly.